.vscode/launch.json

```diff
@@ -15,10 +15,6 @@
       "runtimeExecutable": "${execPath}",
       "args": ["--extensionDevelopmentPath=${workspaceRoot}"],
       "outFiles": ["${workspaceRoot}/client/out/**/*.js"],
-      "preLaunchTask": {
```

Comment:
remove the `preLaunchTask` , since `esbuild --watch` can not be used as `preLaunchTask` with vscode [issue](https://github.com/microsoft/vscode/issues/180641), unless you  install a vscode extension, [esbuild-problem-matcher](https://marketplace.visualstudio.com/items?itemName=connor4312.esbuild-problem-matchers). 
**Pros**
- esbuild is a [blazing fast bundler](https://esbuild.github.io/faq/#benchmark-details), which make vscode extension packaging super fast.
- we could reduce the size of extension, which improving installation speed.

**Cons**
- you need to run a watch task manually during development mode.

Reply:
I think it's too inconvenient to run the watch task manually in dev mode. preLaunch should only affect dev mode, right? The bundling is only needed for publishing the package.

Reply:
It works with `npm run esbuild-base -- --sourcemap --watch &`

Reply:
You still need to install the related `$esbuild-watch` vscode extension to enable problemWatcher, or vscode will warn 
![image](https://github.com/FuelLabs/sway-vscode-plugin/assets/160697994/69da73a6-93fb-4501-b581-3a1d38993875)

Reply:
I have updated the scripts with `&` postfix